apps/examples/package.json

```diff
@@ -39,6 +39,8 @@
 		"@tldraw/dotcom-shared": "workspace:*",
 		"@tldraw/sync": "workspace:*",
 		"@vercel/analytics": "^1.1.1",
+		"ag-grid-community": "^32.1.0",
```

Comment:
this is fine for this PR but i'm wondering in the future, for AG grid and pdf libs, that we delay load them/put them in a separate JS module. might start getting out of hand that the examples app has to get a bunch of code for only one of the examples.

Reply:
we actually already do this automatically! each example is set up to be lazy-loaded when you navigate to it, so AG grid doesn't get loaded until you open the example that uses it

Reply:
ah killer! 🙌